Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -2272,6 +2274,8 @@ public void showMap(final String pureScriptId, boolean isEnableCurrentLocation,
     @JavascriptInterface
     public void mapSnapShot(final String pureScriptId, final String json, final String routeType, final boolean actualRoute, final String callback) {
         try {
+            View view = activity.findViewById(Integer.parseInt(pureScriptId));
```

Comment:
As this view we are not using These two lines can we replaced with the following
`if ((activity.findViewById(Integer.parseInt(pureScriptId)) == null) return;`